app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,78 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      
+      def index
+        @Targets = current_user.targets
+      end
+
+      def create
+        if target_params.nil?
+          render json: { errors: ['The arguments given are incorrect.'] }, status: :bad_request and return 
+        end
+        @target = Target.new(user:current_user,
+          lat: target_params[:lat], 
+          lng: target_params[:lng], 
+          radius: target_params[:radius], 
+          topic: Topic.find(target_params[:topic]))
+
+        if @target.save 
+          @near = get_near(@target)
+          render :newTarget
+        elsif @target.errors[:limit].any?
+          render json: { errors: ['You have exceeded the amount of Targets, please remove one before create a new one.'] }, status: :bad_request 
+        else
+          render json: { errors: ['Could not creat Target'] }, status: :bad_request
+        end
+      end
+
+      def target_params
+        target_params = params.require(:target).permit(:lat, :lng, :radius, :topic, :id)
+      end
+
+      def destroy
+        @target = current_user.targets.find_by_id(params[:id])
+
+        if @target.nil?
+          render json: { errors: ['There is no target with this ID'] }, status: :bad_request and return 
+        end
+        if @target.destroy
+          render :destroy
```

Comment:
no need to specifically render `destroy` when success, it should be the default behaviour.